files/scripts/syncd.sh

```diff
@@ -104,11 +104,12 @@ start() {
         if [[ x"$WARM_BOOT" != x"true" ]]; then
             if [[ x"$(/bin/systemctl is-active pmon)" == x"active" ]]; then
                 /bin/systemctl stop pmon
-                /usr/bin/hw-management.sh chipdown
-                /bin/systemctl restart pmon
-            else
-                /usr/bin/hw-management.sh chipdown
+                debug "pmon is active while syncd starting, stop it first"
             fi
+            /usr/bin/hw-management.sh chipdown
+            debug "Starting pmon service..."
+            /bin/systemctl restart pmon
```

Comment:
Any preference for 'restart' over 'start' ?

Reply:
In this case 'Restarting pmon service...' debug message should better describe the flow

Reply:
No need to use "restart" here. Use 'start' instead.